### PR TITLE
Control Cards on the Bridge UI

### PR DESCRIPTION
**Pull request by AtelyPham**
- feat: add Deposit control card
- chore: add bridging token to deposit control card
- feat: add Withdraw control card
- feat: add AmountMenu component
- add AmountMenu to fixed amount and custom amount inputs
- fix: button style and remove unused file
- add Popover component
- pass props to withdraw card
- add button prop for withdraw card
- feat: add transfer control card

## Summary of changes
- Add Withdraw Control Card
- Add Deposit Control Card
- Add Deposit Control Card
- Add Popover component

### Reference issue to close (if applicable)
- Closes 
	- https://github.com/webb-tools/webb-dapp/issues/593
	- https://github.com/webb-tools/webb-dapp/issues/598

-----
### Code Checklist 

- [x] Tested
- [x] Documented

### Additional info

You can review created cards [here](https://deploy-preview-655--stats-dev.netlify.app/#/bridge-control-showcase).

### CODE REVIEW

**Review by dutterbutter — CHANGES_REQUESTED** [review]
Please fix the typo, should be "paste"

Screen Shot 2022-10-17 at 9 29 32 AM

File: packages/webb-ui-components/src/containers/WithdrawCard/types.d.ts [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+   */
+  relayerInputProps?: ComponentProps<typeof RelayerInput>;
+
+  /**
+   * The recipient input props
+   */
+  recipientInputProps?: ComponentProps<typeof RecipientInput>;
+
+  /**
+   * The recived amount
```
Comment (dutterbutter) [inline packages/webb-ui-components/src/containers/WithdrawCard/types.d.ts:52]:
```suggestion
   * The received amount
```

**Comment by dutterbutter** [comment]
@AtelyPham does this also close #598 ?

**Comment by AtelyPham** [comment]
Yes, I have linked related issues.

**Review by dutterbutter — APPROVED** [review]

**Pull request merged**